### PR TITLE
fix(node) Explain when npm ci is used instead of npm install

### PR DESCRIPTION
**Pull request by johnsudaar**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -26,6 +26,8 @@ services:
       - ./:/usr/src/app
     command: bin/webpack-dev-server --config config/webpack/development.js
     environment:
+      BUNDLE_BIN: "/usr/src/app/vendor/.bundle/ruby/3.0.0/bin"
+      BUNDLE_PATH: "/usr/src/app/vendor/.bundle/ruby/3.0.0"
```
Comment (EtienneM) [inline docker-compose.yml:30]: These are already defined a few lines below
```suggestion
```

**Review by EtienneM — CHANGES_REQUESTED** [review]

**Review by EtienneM — APPROVED** [review]

File: _posts/languages/nodejs/2000-01-01-start.md [hunk 2]
```diff
@@ -26,6 +26,10 @@ If a `yarn.lock` file is present at the root of your project, yarn will be used
 to install the dependencies and run scripts, otherwise npm will build the
 dependencies during the deployment.
 
+If a `package-lock.json` file is present at the root of your project and you
+are using npm version 6 or higher, Scalingo will use `npm ci` to install your
```
Comment (Soulou) [inline _posts/languages/nodejs/2000-01-01-start.md:30]: Using long version `clean-install`  in the doc?

Comment (johnsudaar) [reply _posts/languages/nodejs/2000-01-01-start.md:30]: Everybody use ci in their docs (heroku and NPM themself)

Comment (Soulou) [reply _posts/languages/nodejs/2000-01-01-start.md:30]: Maybe between parenthesis, for isntance I didn't know what was `ci`, I thought weird, Continuous Integration? wtf I had to run `npm ci --help` on my laptop. That's why I'm proposing to explicit what it is like:

```
[...] Scalingo will use `npm ci` (for `clean-install`) to install your [...]
```

**Review by Soulou — APPROVED** [review]

**Pull request merged**